chain/src/chain.rs

```diff
@@ -299,9 +312,9 @@ impl ChainService {
         db_txn.insert_tip_header(&target_tip_header)?;
         db_txn.insert_current_epoch_ext(&target_epoch_ext)?;
 
-        for blk in fork.attached_blocks() {
-            db_txn.delete_block(blk)?;
-        }
```

Comment:
I think previous implementation is wrong? Because we expect `target_tip_hash` is on main_chain, so `fork.attached_blocks()` should be empty, is it right?

Reply:
yes.

If we want to delete blocks, we should use `for blk in fork.detached_blocks()`, but actually we don't need to delete blocks for testing usage, and even worse if we run `sync` at the same time and there are incoming new blocks, a crash will make the db corrupted.

So, I think it's Ok to remove this part.

Reply:
Unrelated to this PR.